### PR TITLE
Added issue template and pull request template

### PR DESCRIPTION
**Pull request by hindu-muppala**
@Vikash-8090-Yadav ,please review and give feedback

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
Please share   the ss of the output of both issue and pr template .

**Comment by hindu-muppala** [comment]
@Vikash-8090-Yadav ,you can able to see preview by visiting my repo.
https://github.com/hindu-muppala/SaL--dApp/blob/issue_template_pr/.github/ISSUE_TEMPLATE (this is for issue template)
https://github.com/hindu-muppala/SaL--dApp/blob/issue_template_pr/.github/pr_request.md  (this is for pr).

If this is not possible,I will put screenshots.

**Comment by hindu-muppala** [comment]
@Vikash-8090-Yadav , is the templates are ok,if not please give feedback I will change.

**Comment by Vikash-8090-Yadav** [comment]
Hey @hindu-muppala  it's good but not  up to the mark. Refer this and made like  this with some changes for the both issue and pr https://github.com/Vikash-8090-Yadav/Future.WebNet/blob/main/.github/pull_request.md

**Comment by hindu-muppala** [comment]
@Vikash-8090-Yadav ,I am not getting what to change. Can I know what ae the improvement that can be possible?

File: .github/pr_request.md [hunk 1]
```diff
@@ -24,11 +24,10 @@ Here comes all the changes proposed through this PR
...
 - [ ] I have updated the documentation accordingly.
 - [ ] All new and existing tests passed.
-- [ ] This PR does not contain plagiarized content.
 - [ ] The title of my pull request is a short description of the requested changes.
-
+- [ ] I'm GSSOC'23 Contributor 
 <!--Add screen shots of the changed output-->
 
-## Screenshots
+## Screenshots | GIF's (Recommended) 📷
```
Comment (Vikash-8090-Yadav) [inline .github/pr_request.md:31]: Instead of the gifs make  Video

File: .github/pr_request.md [hunk 2]
```diff
@@ -24,11 +24,10 @@ Here comes all the changes proposed through this PR
 - [ ] My change requires a change to the documentation.
 - [ ] I have updated the documentation accordingly.
 - [ ] All new and existing tests passed.
-- [ ] This PR does not contain plagiarized content.
 - [ ] The title of my pull request is a short description of the requested changes.
-
+- [ ] I'm GSSOC'23 Contributor
```
Comment (Vikash-8090-Yadav) [inline .github/pr_request.md:28]: Instead of the Guve a heading `Are you  participating in  any open source Programme?`

**Review by Vikash-8090-Yadav — COMMENTED** [review]
what is the use of this box , it's restricting me write any thing .Please check this .
Screenshot from 2023-05-24 01-52-48

**Comment by hindu-muppala** [comment]
@Vikash-8090-Yadav ,I tested we can able to write
image
image
image
image

**Review by Vikash-8090-Yadav — APPROVED** [review]
LGTM

**Pull request merged**